conda-env.yml

```diff
@@ -0,0 +1,17 @@
+name: motile-tracker-conda
+channels:
+  - conda-forge
+  - funkelab
+  - gurobi
+dependencies:
+  - python=3.11.*
+  - pip
+  - gcg
```

Comment:
@cmalinmayor I don't know if you had this problem on windows but until I added these dependencies - it would not find ilpy. I may not need all - I just copied this from the SCIP site

Reply:
I had major issues with pixi and ilpy the other day - I am not sure if this is causing the problem that you are having, but it would be great if you could check:

Since motile_tracker was a pip dependency for me (I wanted editable install), pixi installed all of its dependencies (including motile, including ilpy) from pip. ilpy was then installed twice, once from conda and once from pip, with the pip one taking precedence (there was a warning about this during the pixi install). With the help of [this](https://github.com/prefix-dev/pixi/issues/1537) issue, I solved it by adding a `local_mapping.json file` with `{"ilpy": "ilpy"}` in it, and adding `
conda-pypi-map = {"funkelab" = "local_mapping.json"}` to the project section of my pypi configuration.